modules/android/_preload.lua

```diff
@@ -13,11 +13,12 @@
 --
 
 	p.ANDROID = "android"
-	p.ANDROIDPROJ = "androidproj"
+	p.ANDROIDPROJ = "Packaging"
```

Comment:
there is a test in `test_android_files` that crashes because it uses "androidproj".

is it ok to make it crash for users?

anyways you'll have to change the test to use the correct "Packaging".

Reply:
All right, I'm gonna fix those in a second.